ui/public/index.html

```diff
@@ -29,6 +29,39 @@
     -->
     <title>CERN Library Catalogue</title>
   </head>
+  <!-- Piwik -->
+  <script type="text/javascript">
+    var pkBaseURL =
+      'https:' == document.location.protocol
+        ? 'https://piwik.web.cern.ch/'
+        : 'http://piwik.web.cern.ch/';
+    document.write(
+      unescape(
+        "%3Cscript src='" +
+          pkBaseURL +
+          "piwik.js' type='text/javascript'%3E%3C/script%3E"
+      )
+    );
+  </script>
+  <script type="text/javascript">
```

Comment:
You can remove this double closing/opening of JS.
Can you also add a `if PROD`? We need to find the right variable... otherwise locally it will keep sending requests...